nsqd/topic.go

```diff
@@ -199,8 +199,8 @@ func (t *Topic) PutMessages(msgs []*Message) error {
 		if err != nil {
 			return err
 		}
+		atomic.AddUint64(&t.messageCount, 1)
```

Comment:
instead of single increments, let's maintain a local counter and then perform a single atomic add below on line 203?

Reply:
ahh right, I completely missed the fact that it was returning, too. thx 🙏

Reply:
I have modified my codes according to @ploxiln 's advice. Please take a look 🙂